application/forms/RedisConfigForm.php

```diff
@@ -169,16 +298,22 @@ public function addSubmitButton()
 
     public static function checkRedis($form)
     {
-        $streamy = version_compare(phpversion('redis'), '4.3.0', '>=');
```

Comment:
Why did you remove that and everything related to it? What does this have to do in any way with TLS support?

Restore this. It's purpose is that Icinga DB Web still functions *kind of* on PHP 5.6 installations.

Reply:
But this PR drops PHP-Redis usage?

Reply:
Aye.. didn't notice that. Would have been good if you mentioned this in the OP. It's an architectural change after all.